packages/astro/src/core/ssr/index.ts

```diff
@@ -245,15 +245,23 @@ export async function render(renderers: Renderer[], mod: ComponentInstance, ssrO
 
   // inject CSS
   [...getStylesForURL(filePath, viteServer)].forEach((href) => {
-    tags.push({
-      tag: 'link',
-      attrs: {
-        rel: 'stylesheet',
-        href,
-        'data-astro-injected': true,
-      },
-      injectTo: 'head',
-    });
+    if (mode === 'development' && href.includes('.svelte?svelte&type=style')) {
+      tags.push({
+        tag: 'script',
+        attrs: { type: 'module', src: href },
```

Comment:
Confirmed this fixes styles in `portfolio-svelte` and `framework-svelte` (which currently doesn’t have Svelte styles, but if you add styles this works)